resources/qml/Widgets/CheckBox.qml

```diff
@@ -1,77 +0,0 @@
-// Copyright (c) 2020 Ultimaker B.V.
```

Comment:
I don't think you should delete this file.

Personally I would leave this file in and just make it a default UM.Checkbox. That way you don't have to change all the usage of Cura.Checkbox in the rest of the code ;) (And, although not an argument in this case, it doesn't break the API/SDK)

Reply:
I think it is better to have one source of truth for the styling of checkboxes, although I agree that it would be nicer to not have to change the usage of Cura.CheckBox, it could lead to de-syncing of the CheckBox style between all menus (Defined in Cura) and the settingVisibilityMenu.qml (Defined in Uranium). 

I'll keep in mind the API/SDK breaks in the future for non major releases. More than happy to make this change if you believe this would make more sense.